angles/CMakeLists.txt

```diff
@@ -13,12 +13,13 @@ endif()
 
 find_package(ament_cmake REQUIRED)
 
-install(DIRECTORY include/${PROJECT_NAME}/
-        DESTINATION include/${PROJECT_NAME}/
-        FILES_MATCHING PATTERN "*.h")
+add_library(angles INTERFACE)
+target_link_libraries(angles INTERFACE
+  "$<INSTALL_INTERFACE:install/angles>")
+install(TARGETS angles EXPORT export_angles)
 
-include_directories(include)
+ament_export_targets(export_angles)
 
-ament_export_include_directories(include)
+install(DIRECTORY include/ DESTINATION include/angles)
```

Comment:
should be

```
install(DIRECTORY include/ DESTINATION include)
```

Reply:
> @sloretz This fix has caused the header file to be installed in include/angles/angles/angles.h and causes packages including angles/angles.h to fail with file not found since it is installed in the wrong place

This change is intentional, see ros2/ros2#1150 for why. This PR also removed the old-style CMake variable `angles_INCLUDE_DIR`. If one of your packages isn't building, that's probably why. The correct fix is to link against the exported interface target

```CMake
# If your target includes angles headers in it's public headers use this:
target_link_libraries(mylibrary PUBLIC angles::angles)
# If your target only includes angles headers in source files or non-installed headers, use this:
target_link_libraries(mylibrary PRIVATE angles::angles)
```

Reply:
ok thank you